guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -16,6 +16,10 @@ ifdef::satellite[]
 This is required to install the `insights-client` package on hosts.
 endif::[]
 include::snip_prerequisite-configured-smart-proxy-registration-provisioning.adoc[]
+* If your {ProjectServer} or {SmartProxyServer} is behind an HTTP proxy, configure the Subscription Manager on your host to use the HTTP proxy for connection.
```

Comment:
Subcription Manager is only on RHEL hosts, shoud that be accounted for somehow?

Reply:
It seems that the Global registration template always uses subscription-manager, so this is probably not the issue.
Perhaps the question rather is whether the Global registration currently supports OSs without subscription-manager?

@maximiliankolb @ekohl Can you please clarify this?

Reply:
@adamlazik1 orcharhino Clients for Amazon Linux, CentOS 7, Debian, Oracle Linux, SUSE Linux Enterprise Server, and Ubuntu also contain a packaged `subscription-manager` + dependencies. AlmaLinux, CentOS Stream, Red Hat Enterprise Linux, and Rocky Linux provide the package and its dependencies from the default OS repositories.

Reply:
As @maximiliankolb said, it is available on many platforms, but IMHO it is only relevant for the Katello use case.

I do wonder when it's needed. The way it's designed is that you have either:

Client: subscription-manager -> Foreman

Or:

Client: subscription-manager -> Content Proxy: Apache -> Foreman

How often is there a HTTP proxy in that first step? And how well does it work client certificates? From an architecture perspective I question if is a good idea. Deploying an additional content proxy where clients don't need HTTP proxies is IMHO better, but I also know that in practice some companies have crazy policies.

Reply:
Oh, right, yes, it's a Katello use case. And yes, it looks strange to me to from the architecture perspective, but we have a customer case based on which it was requested, so... this is just in case the customer has a crazy policy in place. :)